<launch>
  <arg name="camera_frame" />
  <arg name="camera_topic" />
  <arg name="use_sim_time" />
  <node pkg="apriltag_ros" exec="apriltag_node" name="apriltag" output="screen">
    <param from="$(find-pkg-share lunabot_config)/config/apriltags/apriltags.yml" />
    <remap from="image_rect" to="$(var camera_topic)"/>
    <param name="use_sim_time" value="$(var use_sim_time)"/>
  </node>
</launch>